src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const act of actions) {
+    switch (act.type) {
+      case 'clear':
+        Object.keys(state).forEach(key => {
```

Comment:
use `for in` to iterate over objects

Reply:
ok)super thanks again. like this comment.  it's not for lazy ones.